web-starter/templates/composer.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "<%- name %>",
+    "type": "project",
+    "license": "GPL-2.0+",
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
+    "require": {
```

Comment:
Need to add `"php": "^5.6 || ^7.0"`. We might want to run PHP 7 locally (hence the `^7.0` version specifier), but since Jenkins runs PHP 5.6, we can't deploy anything that depends on PHP 7.